### PR TITLE
qt: Periodic transifex update

### PR DESCRIPTION
**Pull request by laanwj**
Updates to and from Transifex. Pulls in the following new languages:
- `af` Afrikaans
- `es_419` Spanish (Latin America)
- `es_AR` Spanish (Argentina)
- `es_CO` Spanish (Colombia)
- `fil` Filipino
- `it_IT` Italian (Italy)
- `ro` Romanian
- `sr@latin` Serbian (Latin)
- `ta` Tamil
- `uz@Latn` Uzbek (Latin)
- `zh_HK` Chinese (Hong Kong)

Doing this as an pull request instead of an auto-commit because I'm not sure about some of the new languages. To be specific I had to delete a fake "en_US" locale, which turned out to be Spanish. If someone with the appropriate language skills can at least glance over it that'd be appreciated!

### CODE REVIEW

**Comment by fanquake** [comment]
utACK

**Comment by MarcoFalke** [comment]
Looks like those are just 1:1 copies of their already translated sisters.

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -33,6 +37,7 @@ QT_TS = \
   qt/locale/bitcoin_eu_ES.ts \
   qt/locale/bitcoin_fa_IR.ts \
   qt/locale/bitcoin_fa.ts \
+  qt/locale/bitcoin_fil.ts \
```
Comment (MarcoFalke) [inline src/Makefile.qt.include]: This is Filipino language with Finnish content?

Comment (MarcoFalke) [inline src/Makefile.qt.include]: No, sorry. Both are correct.

**Comment by MarcoFalke** [comment]
ACK 4cbe05b

**Pull request merged**